utils/build/docker/ruby/install_ddtrace.sh

```diff
@@ -8,7 +8,8 @@ if [ -e "/binaries/dd-trace-rb" ]; then
     echo "gem 'ddtrace', require: 'ddtrace/auto_instrument', path: '/binaries/dd-trace-rb'" >> Gemfile
 elif [ $(ls /binaries/ruby-load-from-bundle-add | wc -l) = 0 ]; then
     echo "Install prod version"
-    echo "gem 'ddtrace', '~> 1.0.0.beta1', require: 'ddtrace/auto_instrument'" >> Gemfile
+    echo "1.8.0"
+    echo "gem 'ddtrace', '~> 1.8.0', require: 'ddtrace/auto_instrument'" >> Gemfile
```

Comment:
Can you automate this to "latest stable release" instead?
IIRC, @lloeki was saying it was possible when for example removing the version number.